packages/editor/src/components/block-list/block.js

```diff
@@ -405,9 +405,12 @@ export class BlockListBlock extends Component {
 		const shouldShowInsertionPoint = ( isPartOfMultiSelection && isFirst ) || ! isPartOfMultiSelection;
 		const canShowInBetweenInserter = ! isEmptyDefaultBlock && ! isPreviousBlockADefaultEmptyBlock;
 
+		// We allow an invalid block to be re-edited in HTML mode
+		const isValidMode = isValid || mode === 'html';
```

Comment:
Minor: the variable name suggests that it checks if the mode is valid or not, while the variable contains if the block is valid or we are in HTML mode. Maybe we can call it isValidOrModeIsHtml.

Reply:
Would it be <code><var>isValidOrModeIsH<b>TML</b></var></code> or  <code><var>isValidOrModeIsH<b>tml</b></var></code>? I forgot what the consensus was for acronyms in variable names. (I know <i>id</i> should be `id` if at the start or `Id` otherwise, since it is not an acronym but just an abbreviation like <i>var</i> is an abbreviation of <i>variable</i>.)